examples/SampleApp/Gemfile

```diff
@@ -1,9 +1,9 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby '2.7.5'
+ruby '>= 2.6.10'
```

Comment:
why the downgrade? 

2.7.5 comes from here I think

https://github.com/react-native-community/react-native-template-typescript/blob/main/template/_ruby-version